frontend-ui/src/constants.ts

```diff
@@ -9,6 +9,7 @@ export default {
   COOKIE_LIFETIME_EXPIRY: '10y', // 10 years
   TASKS_LOAD_SCHEDULES_DELAY: 100,
   MAX_SCHEDULES_IN_SELECTION_REQUEST: 200,
+  KIWIX_DOWNLOAD_URL: 'https://download.kiwix.org/zim',
```

Comment:
Rename to `ZIM_DOWNLOAD_URL` + move to `config.json` so that we can customize it per deployment (value is different between farm.openzim.org and farm.zimit.kiwix.org typically)